packages/ui/src/message-room-details-dialog/MessageRoomDetailsDialog.jsx

```diff
@@ -252,8 +252,40 @@ function MembersPanel({ config }) {
     addmembersevent(values);
   };
 
+  //to remove member from the channel
+  //the id was passed from the RemoveMemberModal component to be able to identify the id we want to remove
+  // when we destructure the member we want to remove
+  //this  removememberevent(id); is just to console that the plugin/member has being delete
```

Comment:
Comments that document what a function does should be written with jsdoc so it actually helps your IDE intellisense